generate_table.Rmd

```diff
@@ -0,0 +1,624 @@
+---
+title: "Generate Table from XML Files"
+author: "Quinn White"
+date: "`r Sys.Date()`"
+output:
+  rmdformats::readthedown:
+    df_print: paged
+    code_folding: hide
+---
+
+```{r setup, include=FALSE}
+knitr::opts_chunk$set(echo = TRUE, message=FALSE, warning=FALSE)
+```
+
+```{css,echo=FALSE}
+
+#sidebar h2 { background: #3B5561 !important;
+}
+#main .nav-pills > li.active > a,
+#main .nav-pills > li.active > a:hover,
+#main .nav-pills > li.active > a:focus {
+    background-color: #22983B;
+}
+
+#main .nav-pills > li > a:hover {
+    background-color: #22983B;
+}
+
+h1, h2, h3, h4, h5, h6, legend {
+    color: #334B56;
+}
+
+#nav-top span.glyphicon {
+    color: #22983B;
+}
+
+
+
+#table-of-contents h2 {
+    background-color: #3B5561;
+}
+
+#content{
+max-width:2300px;
+}
+
+
+
+```
+
+
+```{r}
+
+library(tidyverse)
+library(xml2)
+library(kableExtra)
+library(lubridate)
+library(forcats)
+
+
+```
+
+# Generating Table for All Files
+
```

Comment:
Absolutely fantastic work done here, Quinn!!! The only comments I include are for readability of code. Suuuuper minor, I'm wondering if this initial chunk could be rearranged a bit, so the function definition is by itself, then the variable creation (files, variables_to_extract) and applying get_df all happen in the same place?